pkg/storj/segment.go

```diff
@@ -17,6 +17,16 @@ type SegmentListItem struct {
 // SegmentDownloadInfo represents segment download information inline/remote
 type SegmentDownloadInfo struct {
```

Comment:
I'm not quite sure how much this makes sense since you also need the addressed order limits for this to work... but I guess it's fine for now.

Reply:
Because addressed order limits are from `pb` I'm returning them as a separate value in `client.DownloadSegment`